books/free-programming-books-langs.md

```diff
@@ -1258,6 +1259,12 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [The Complete Redux Book](https://leanpub.com/redux-book) - Boris Dinkevich and Ilya Gelman  *(Leanpub account or valid email requested)*
 
 
+#### Svelte
+
+* [Learn the Svelte JavaScript Framework](https://www.youtube.com/watch?v=ujbE0mzX-CU) - Noah Glaser, freeCodeCamp (YouTube)
```

Comment:
not a book

Reply:
Sorry, I must've misunderstood [the previous comment](https://github.com/EbookFoundation/free-programming-books/pull/5909#discussion_r721835432). Where would be a good place to sort a 7hr video course?

Reply:
courses